main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+        for player_name, player_info in data.items():
+            race = player_info.get("race")
+            guild = player_info.get("guild")
+            skills = race.get("skills")
+
+            race, _ = Race.objects.get_or_create(
+                name=race["name"],
+                description=race["description"]
```

Comment:
Race can be None due to .get method use player_info["race"] or .get but with default argument as dict
The same with skill and player